common/configdb.h

```diff
@@ -71,10 +71,14 @@ class ConfigDBConnector_Native : public SonicV2Connector_Native
             return self.getDbName()
 
         ## Note: callback is difficult to implement by SWIG C++, so keep in python
-        def listen(self):
+        def listen(self, start=True):
```

Comment:
If i understand correctly, to use ConfigDBConnector without blackout, the listen() has to be called first with start=False arg, fill the cache and operate upon initial data and then call process() with the filled cache.  

If that is indeed the case, please comment this behavior and how to use this class and listen/process methods to avoid blackout.

Reply:
Sure